### PR TITLE
Remove jQuery

### PR DESCRIPTION
**Pull request by as-cii**
This pull request replaces atom-space-pen-views with etch and atom-select-list. Feature-wise nothing should change, but this yet another step toward removing jQuery from Atom Core.

/cc: @ungb for 👀. @simurai: since you're one of the top-3 committers of this package, could you take a look as well and help me ensure I haven't introduced any regression, please? ❤️

### CODE REVIEW

File: lib/styleguide-view.js [hunk 1]
```diff
@@ -0,0 +1,1267 @@
...
+
+          <div className='styleguide-controls btn-group'>
+            <button className='btn' onclick={() => this.collapseAll()}>Collapse All</button>
+            <button className='btn' onclick={() => this.expandAll()}>Expand All</button>
+          </div>
+        </header>
+
+        <main className='styleguide-sections'>
+          <StyleguideSection onDidInitialize={this.didInitializeSection.bind(this)} name='variables' title='Variables'>
+            <p>Use these UI variables in your package's stylesheets. They are set by UI themes and therefore your package will match the overall look. Make sure to @import 'ui-variables' in your stylesheets to use these variables.</p>
```
Comment (simurai) [inline lib/styleguide-view.js]: Maybe one thing. The `'` in `package's` messes with the syntax highlighting of this file. 

screen shot 2017-01-06 at 1 48 00 pm

Also here on github.com.

Should we just use `&apos;` instead?

Comment (as-cii) [inline lib/styleguide-view.js]: I think not escaping punctuation helps readability quite a lot: to circumvent this issue (which I believe is due to the language-javascript package) I have installed the `language-babel` package. That said I am totally cool to switch to HTML entities if you think that'd be better. 👍

Comment (50Wliu) [inline lib/styleguide-view.js]: Yeah.  It looks like you're using JSX (or something similar?) here, which language-javascript doesn't support as it isn't native JS.  To get it to render correctly on GitHub I think you'll need to switch the extension to `.jsx`.

Comment (simurai) [inline lib/styleguide-view.js]: Yes, `language-babel` fixes it. Ok, let's keep as is. I should probably keep that package enabled anyways.

**Comment by ungb** [comment]
@as-cii I was able to test this package as well as others that has changes to remove atom-space-pen-views/jquery.  Everything looks good to me.

**Pull request merged**